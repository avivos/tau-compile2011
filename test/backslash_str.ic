String d = "my name is ben\james, means you can call me either of the two"