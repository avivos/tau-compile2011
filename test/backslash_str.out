1: CLASS_ID(String)
1: ID(d)
1: ASSIGN
Line 1: illegal literal inside a string: \
